src/dolphin/atmosphere/troposphere.py

```diff
@@ -135,10 +135,11 @@ def estimate_tropospheric_delay(
     tropo_height_levels = np.concatenate(([-100], np.arange(0, 9000, 500)))
 
     for key in slc_files:
-        if len(key) == 1:
-            first_date = key
+        if "compressed" not in str(slc_files[key][0]).lower():
+            one_of_slcs = slc_files[key][0]
             break
-    wavelength = oput.get_radar_wavelength(slc_files[first_date][0])
+
+    wavelength = oput.get_radar_wavelength(one_of_slcs)
```

Comment:
Look like this is all done becuase the compressed SLC does not have all the metadata. I think we need to revise compressed SLC as this issue will come up again when want to make baseline time-series. For now its fine to keep going.

Reply:
yes thank you, I opened an issue to remember doing it later